docs/06-contributors/99-maintainers/050-operations.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Operations Overview
+id: operations
+keywords: [operations]
+---
+
+## Coverage
+
+"Wing" encompases several projects/sites.
+This document covers operations for the following:
+
+- https://docs.winglang.io (https://github.com/winglang/docsite)
+- Wing (https://github.com/winglang/wing)
+- Wing Console (https://github.com/winglang/console)
+
+## Severity Levels
+
+Incidents are classified by severity level.
+The severity level is determined by the impact of the incident on user experience.
+
+| Severity | Description | Example |
+| - | - | - |
+| P0 | Extreme user interruption, no reasonable workaround | Unable to install Wing |
+| P1 | Major degradation of user experience, possible workarounds | Main build is broken in CI (cannot publish) |
+| P2 | The system is usable but has a minor issue. | Error message regression, new message is very confusing |
+
+## Response Process
+
+### Acknowledgement
+
+Acknowledgement signifies that someone is taking responsibility for the incident.
+This does not mean that the person acknowledging the incident is who will be resolving it, but they will at least be responsible for triage and communication.
+
+Depending on the issue type, the acknowledgement may be automated through tooling (e.g. BetterUptime).
+If not automated, the responder should create a thread in the #alert channel in Slack to make sure others are aware.
+
+### Response
+
+If possible, immediate action should be taken to mitigate the issue.
+A runbook should be followed if one exists.
+Otherwise, the responder should take the best action they can to mitigate the issue.
+For high severity issues based on regressions, the default behavior should be to revert back to the last known good version.
+
+Another good response is to reach out to someone who has more information about the issue.
+This could be the person who wrote the code, or someone who has more context about the system.
```

Comment:
Maybe we can mention to be mindful of the timezones different maintainers are working in by check their Slack profile to see what their local time is

Reply:
I imagine the mindfulness changes with priority level of the issue. IE if wing cannot be installed, and the responder does not have the ability to quickly resolve the issue then is it acceptable to wake someone up in the middle of the night? (probably)